.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,32 @@
+Thanks for opening an issue! A few things to keep in mind before moving forward:
+
+  - Please note that an issue is not the best place to ask questions. Using [Stack Overflow](https://stackoverflow.com/questions/tagged/rocket.chat) or [Server Fault](https://serverfault.com/search?q=Rocket.Chat) for that is more appropriate. Also don't forget to search our [Documentation](https://rocket.chat/docs/)
+
+  - Please [search our issues](https://github.com/RocketChat/Rocket.Chat/issues) for an existing issue before opening a new one. This helps us keep all information we need to fix the issue in one place.
+
+  - Before reporting a bug, please try reproducing your issue against the latest version of Rocket.Chat and don't forget to include any errors on your console (browser and server).
```

Comment:
Might include a link to this so those unfamiliar can figure out how to get logs: https://rocket.chat/docs/contributing/reporting-issues#gathering-logs